Logging

Paola351 opened the pull request:
added logger configuration file

Paola351 committed:
added validations

Paola351 committed:
added logging

Paola351 committed:
fixed checks

Paola351 committed:
Merge branches 'logging' and 'main' of https://github.com/Paola351/recurring-tasks into logging

 Conflicts:
	checkTasks.py

Paola351 committed:
fixed comments

Paola351 committed:
Merge branches 'logging' and 'main' of https://github.com/Paola351/recurring-tasks into logging

 Conflicts:
	checkTasks.py

Paola351 committed:
merged with master

Paola351 committed:
merged from upstream

Paola351 committed:
fixed log message

Paola351 committed:
Merge remote-tracking branch 'origin/logging' into logging

Paola351 committed:
added logfile

error401de committed:
Update tasks.ini

error401de committed:
Update tasks.ini

error401de committed:
Update tasks.ini

Paola351 committed:
added loggerfile

Paola351 committed:
Merge remote-tracking branch 'origin/logging' into logging

Paola351 committed:
Merge branch 'main' of https://github.com/error401de/recurring-tasks into logging

 Conflicts:
	checkTasks.py

Paola351 committed:
merged from original main

Paola351 committed:
fixed indentation

Paola351 commented:
Don't forget the labels please :D

error401de reviewed (changes requested):
The requested change refers to an old state of the script, so it would override a lot.
Can you please make sure to edit the PR accordingly (or create an additional, rebased one with only the relevante commit)
Btw I added the hacktoberfest-accepted label to your already merged PRs ;)

Paola351 committed:
added logging through logger.ini

Paola351 commented:
> The requested change refers to an old state of the script, so it would override a lot.
> Can you please make sure to edit the PR accordingly (or create an additional, rebased one with only the relevante commit)
> Btw I added the hacktoberfest-accepted label to your already merged PRs ;)

Done: https://github.com/error401de/recurring-tasks/pull/18
